tests/serializers/tests.py

```diff
@@ -411,6 +411,187 @@ def test_indentation_whitespace(self):
             if re.search(r'.+,\s*$', line):
                 self.assertEqual(line, line.rstrip())
 
+    def test_helpful_error_message_invalid_pk(self):
+        """
+        #19820 - If there is an invalid primary key, expect that the error message
```

Comment:
not sure there is a need to include the ticket in every test.
Instead of "expect that", I would phrase it as "the error message should tell ..."